Packs/Sixgill-Darkfeed/Integrations/Sixgill_DVE_Feed/Sixgill_DVE_Feed.py

```diff
@@ -0,0 +1,210 @@
+        ("NVD - publication date", "nvdpublicationdate"),
+        ("CVSS 2.0 score", "cvss20score"),
+        ("CVSS 2.0 severity", "cvss20severity"),
+        ("NVD Vector - V2.0", "nvdvectorv20"),
+        ("NVD Vector - V3.1", "nvdvectorv31"),
+    ]
+)
+
+
+""" HELPER FUNCTIONS """
+
+
+def module_command_test(*args):
+    """
+    Performs basic Auth request
+    """
+    response = SESSION.send(
+        request=SixgillAuthRequest(
+            demisto.params()["client_id"], demisto.params()["client_secret"], CHANNEL_CODE
+        ).prepare(),
+        verify=VERIFY,
+    )
+    if not response.ok:
+        raise Exception("Auth request failed - please verify client_id, and client_secret.")
+    return "ok", None, "ok"
+
+
+def get_description(fileds_obj):
+    description_string = ""
+    for name, sixgill_name in DESCRIPTION_FIELD_ORDER.items():
+        description_string += f"{name}: {fileds_obj.get(sixgill_name)}\n"
+    fileds_obj["description"] = description_string
+    return fileds_obj
+
+
+def create_fields(stix_obj, event_obj, nvd_obj, score_obj, ext_id):
+    fields = {}
+    try:
+        fields = {
+            "description": "",
+            "creationdate": stix_obj.get("created", ""),
+            "modified": stix_obj.get("modified", ""),
+            "externalid": ext_id,
+            "sixgilldvescorecurrent": score_obj.get("current", ""),
+            "sixgilldvescorehighesteverdate": score_obj.get("highest", {}).get("date", ""),
+            "sixgilldvescorehighestever": score_obj.get("highest", {}).get("value", ""),
+            "sixgillpreviouslyexploitedprobability": score_obj.get("previouslyExploited", ""),
+            "eventname": event_obj.get("name", ""),
+            "eventtype": event_obj.get("type", ""),
+            "eventaction": event_obj.get("action", ""),
+            "previouslevel": event_obj.get("prev_level", ""),
+            "eventdescription": event_obj.get("description", ""),
+            "eventdatetime": event_obj.get("event_datetime", ""),
+            "cvss31score": nvd_obj.get("base_score_v3", ""),
+            "cvss31severity": nvd_obj.get("base_severity_v3", ""),
+            "nvdlink": nvd_obj.get("link", ""),
+            "nvdlastmodifieddate": nvd_obj.get("modified", ""),
+            "nvdpublicationdate": nvd_obj.get("published", ""),
+            "cvss20score": nvd_obj.get("score_2_0", ""),
+            "cvss20severity": nvd_obj.get("severity_2_0", ""),
+            "nvdvectorv20": nvd_obj.get("vector_v2", ""),
+            "nvdvectorv31": nvd_obj.get("vector_v3", ""),
+        }
+    except Exception as err:
+        demisto.error(err)
+        demisto.error(traceback.format_exc())
+    return fields
+
+
+def stix_to_indicator(stix_obj, tags: list = [], tlp_color: Optional[str] = None):
+    indicator: Dict[str, Any] = {}
+    try:
+        ext_obj = stix_obj.get("external_references", [])
+        ext_id = ""
+        if ext_obj[0]:
```

Comment:
This line will cause a 'list index out of range' error when ext_obj is an empty list.
I think it is better to do something like:
```suggestion
        if ext_obj and ext_obj[0]:
```